flocker/control/_model.py

```diff
@@ -930,6 +939,42 @@ def get_information_wipe(self):
         return _WipeNodeState(node_uuid=self.uuid, attributes=attributes)
 
 
+@implementer(IClusterStateChange)
+class UpdateNodeStateEra(PClass):
+    """
+    Update a node's era.
+
+    :ivar UUID uuid: The node's UUID.
+    :ivar UUID era: The node's era.
+    """
+    uuid = field(type=UUID, mandatory=True)
+    era = field(type=UUID, mandatory=True)
+
+    def update_cluster_state(self, cluster_state):
+        """
+        Record the node's era and discard the ``NodeState`` if it doesn't
+        match the era.
+        """
+        if cluster_state.node_uuid_to_era.get(self.uuid) != self.era:
+            # Discard the NodeState:
+            cluster_state = cluster_state.set(
+                nodes={n for n in cluster_state.nodes
+                       if n.uuid != self.uuid})
+        cluster_state = cluster_state.transform(
+            ["node_uuid_to_era", self.uuid], self.era)
+        return cluster_state
+
+    def get_information_wipe(self):
+        """
+        Do nothing: no followup wipes are necessary.
+
+        If we did discard the node state we'll be leaving its wiper hanging,
+        but it won't actually do anything if it can't find the node so
+        that's OK.
```

Comment:
I've read this sentence through a few times now and don't really know what it says. Perhaps it relies on some global knowledge of the plan that can't be readily inferred from the surround code? 

Unfortunately, because I can't yet puzzle it out, I also can't give any concrete suggestions for improvement. 

Anyway, I'd appreciate it if you could have a shot at rephrasing to make it more clear.